webapp/src/index.css

```diff
@@ -50,37 +50,37 @@ body {
 }
 
 .heading-bold {
-  font-size: 1.5rem;
+  font-size: 2rem;
```

Comment:
Adjust text sizes, in desktop where to small

Reply:
The font styles match the mobile text styles from Figma with a base of 16px.
What do you think about using a media query for desktop and changing the base font-size?

Reply:
https://github.com/42AGV/ft_transcendence/issues/101

I think finally the best option is to update everything in this task, thanks for the observation!

Reply:
Thank you, good catch! We could use classes instead of hardcoding the sizes in the Text component.

If the sizes are small only for the desktop, we could use media queries and match the values with the ones we have for desktop text in Figma.

Either setting a bigger base font
```
@media (min-width: 768px) {
  :root {
    font-size: 20px 
  }
}
```


Or setting the desktop size in each text class

```
@media (min-width: 768px) {
  .title-bold {
    font-size: 3rem;
  }
   
  .subtitle-bold {	  
    font-size: 2.5rem;	    
  }

  ...
}
```

And if the sizes are smaller for both mobile and desktop, we could update both default sizes in the index.css and Figma, and I think we also have to use media queries in this case.

Reply:
Nice!! If you want, you can add that comment to the issue so this info don't get lost